.github/workflows/release_github.yml

```diff
@@ -71,3 +71,38 @@ jobs:
               --title="Release ${{ steps.bump.outputs.RELEASE_TAG }}" \
               --generate-notes \
               --draft
+
+  tidy_workspace:
+    # only run if action above succeeds
+    needs: github_release
+    runs-on: "ubuntu-latest"
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_PAT }}
+
+      - name: Configure git
+        run: |
+          git config user.email "actions@github.com"
+          git config user.name "GitHub Actions"
+          git pull
+
+      - name: Merge updates into develop
+        run: |
+          git switch test_protected_develop
```

Comment:
I guess. I am struggling to think of a good variable name* for these other than `main`/`develop`, which makes me think that it might be better to just keep them as they are, especially as these are highly conventional names and are unlikely to change at any point.

*: The best I can come up with is  `DEFAULT_GITHUB_BRANCH` and `DEFAULT_RELEASE_BRANCH`.

Reply:
I agree that long variable names are less legible than `main` and `develop`. Let's keep those.